### PR TITLE
[16][MIG] account_move_base_import

### PR DESCRIPTION
**Pull request by florian-dacosta**
Standard migration

### CODE REVIEW

**Review by rvalyi — APPROVED** [review]

**Comment by florian-dacosta** [comment]
Hello @hildickethan-S73 
Since you reviewed the 14.0 one, I wonder if would have time to review this one!

File: account_move_base_import/models/account_journal.py [hunk 1]
```diff
@@ -0,0 +1,396 @@
...
+        move = move_obj.create(move_vals)
+        try:
+            # Record every line in the bank statement
+            move_store = []
+            for line in result_row_list:
+                parser_vals = parser.get_move_line_vals(line)
+                values = self.prepare_move_line_vals(parser_vals, move)
+                move_store.append(values)
+            # move_line_obj.with_context(check_move_validity=False).create(move_store)
+            move_store += self._get_extra_move_line_vals_list(parser, move)
```
Comment (hildickethan) [inline account_move_base_import/models/account_journal.py]: i would just delete the line instead of commenting, but not particularly a big deal

**Review by hildickethan — APPROVED** [review]
lgtm

**Comment by bguillot** [comment]
Thanks

**Pull request merged**